cdc/scheduler/internal/v3/replication/replication_set.go

```diff
@@ -153,6 +153,36 @@ func NewReplicationSet(
 			CheckpointTs: checkpoint,
 			ResolvedTs:   checkpoint,
 		},
+		// We need to initialize the stats with the checkpoint ts.
```

Comment:
How about extract this part as a method, to make the current method not too long, hard to read, since the screen is small.